Neos.Eel/Classes/Helper/DateHelper.php

```diff
@@ -57,6 +71,31 @@ public function format($date, $format)
     }
 
     /**
+     * Format a date to a string with a given cldr format
+     *
+     * @param \DateTime $date
+     * @param string $cldrFormat Format string in CLDR format (see http://cldr.unicode.org/translation/date-time)
+     * @param null|string $locale String locale - example (de|en|ru_RU)
+     *
```

Comment:
We tried adding the tests, but there are like 5 methods to mock for 3 lines of actual code, don't know if it's worth to load Ivan with such task :)